### PR TITLE
Added No Seeb Operators

### PR DESCRIPTION
**Pull request by Andy-Python-Programmer**
Added No Seeb Operators

### CODE REVIEW

File: lib/runtime/runtime.dart [hunk 1]
```diff
@@ -1615,6 +1615,18 @@ Future<ASTNode> visitBinaryOp(Runtime runtime, ASTNode node) async {
       }
       break;
 
+    case TokenType.TOKEN_NOSEEB_OPERATOR:
+      {
+        if (left.type == ASTType.AST_NULL){
+          return right;
+        }
+
+        else{
```
Comment (Calamity210) [inline lib/runtime/runtime.dart:1624]: Else clause is not needed due to the `return`

File: lib/runtime/runtime.dart [hunk 2]
```diff
@@ -1615,6 +1615,18 @@ Future<ASTNode> visitBinaryOp(Runtime runtime, ASTNode node) async {
       }
       break;
 
+    case TokenType.TOKEN_NOSEEB_OPERATOR:
+      {
+        if (left.type == ASTType.AST_NULL){
```
Comment (Calamity210) [inline lib/runtime/runtime.dart:1620]: Prefer `left is NullNode`

**Review by Calamity210 — CHANGES_REQUESTED** [review]

File: lib/runtime/runtime.dart (hunk 2 above)
Comment (Andy-Python-Programmer) [reply lib/runtime/runtime.dart:1620]: Ah. I see. I will do it :)

File: lib/runtime/runtime.dart [hunk 3]
```diff
@@ -686,6 +686,16 @@ Future<ASTNode> visitVarMod(Runtime runtime, ASTNode node) async {
           }
           break;
 
+        case TokenType.TOKEN_NOSEEB_ASSIGNMENT:
+          {
+            if (astVarDef.variableValue is NullNode){
+              return astVarDef.variableValue = value;
```
Comment (Calamity210) [inline lib/runtime/runtime.dart:692]: Be sure to use cascades. ```
 return astVarDef..variableValue = value;
```

File: lib/runtime/runtime.dart [hunk 4]
```diff
@@ -686,6 +686,16 @@ Future<ASTNode> visitVarMod(Runtime runtime, ASTNode node) async {
...
+            }
+
+            return value;
```
Comment (Calamity210) [inline lib/runtime/runtime.dart:695]: Why are you returning value here? It should be astVarDef

**Review by Calamity210 — CHANGES_REQUESTED** [review]

**Review by Calamity210 — APPROVED** [review]

**Pull request merged**